tests/integration/backup/simple/import_test.go

```diff
@@ -151,3 +151,56 @@ func TestBackupImport_WithMultipleNoKeys_NoError(t *testing.T) {
 
 	executeTestCase(t, test)
 }
+
+func TestBackupImport_EmptyObject_NoError(t *testing.T) {
+	test := testUtils.TestCase{
+		Actions: []any{
+			testUtils.BackupImport{
+				ImportContent: `{"User":[{}]}`,
+			},
+			testUtils.Request{
+				Request: `
+					query  {
+						User {
+							name
+						}
+					}`,
+				Results: []map[string]any{
+					{
+						"name": nil,
+					},
+				},
+			},
+		},
+	}
+
+	executeTestCase(t, test)
+}
+
+func TestBackupImport_WithMultipleNoKeysAndInvalidField_NoError(t *testing.T) {
```

Comment:
suggestion: Change function name to end with `_ReturnError` as that is what is expected of the function under test.

Reply:
My bad, will change, thanks

- [ ] Fix test name